javascript/javascript2/week1/spiritAnimal.js

```diff
@@ -0,0 +1,47 @@
+// Getting reference to the body and assigning to the variable below
+const bodyRef = document.querySelector("body");
+
+// Creating the input and appending to the body
+const input = document.createElement("input");
+bodyRef.appendChild(input);
+const btn = document.createElement("button");
+bodyRef.appendChild(btn);
+btn.innerText = "Get spirit animal";
+
+// Creating a tag to display the spirit animal into
+const output = document.createElement("p");
+
+bodyRef.appendChild(output);
+
+let userName = "";
+
+const spiritAnimals = [
```

Comment:
That`s fine. You should create a random phrase here. But you created this list yourself.
const nouns = [
    "wolf",
    "cat",
    "eagle",
    "pinguin",
    "hamster",
    "panda",
    "lion",
    "tiger",
    "puma",
    "zebra",
  ];
  const adjectives = [
    "brave",
    "lazy",
    "fast",
    "sleepy",
    "tired",
    "fluffy",
    "cute",
    "funny",
    "chubby",
    "crazy",
  ];

let randomPhrase = adjectives [Math.floor(Math.random() * adjectives .length)] + " " + nouns [Math.floor(Math.random() * nouns .length)];